docs/Manual_Install.md

```diff
@@ -270,3 +247,21 @@ Gemini and Memento can run integrated into the Sprout nodes, or they can be run
 To install Gemini or Memento as a standalone server, follow the same process as installing a Sprout node, but don't add them to the existing Sprout DNS cluster.
 
 The `sprout_hostname` setting in `/etc/clearwater/shared_config` on standalone application servers should be set to the cluster of the standalone application servers, for example, `memento.cw-ngv.com`.
+
+### I-CSCF configuration
```

Comment:
Should we: 

- make it more explicit that this is optional, and the default is great?
- maybe move this to a separate document (c.f. IBCF.md where we describe that optional function?)